apps/backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "oko-wallet-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node-dev --files --quiet src/index.ts",
+    "start-prod": "node dist/index.js",
+    "backend": "node dist/index.js",
+    "backend:dev": "ts-node-dev --files --quiet src/index.ts",
```

Comment:
why do we need `start`/`backend:dev` and `start-prod`/`backend` if they are the same?